src/ansible/roles/ipa/tasks/main.yml

```diff
@@ -39,27 +39,39 @@
     ipa automountmap-del default auto.direct
   args:
     stdin: '{{ service.ipa.password }}'
+  register: automountmapdel
+  failed_when:
+    - 'automountmapdel.rc != 0 and "automount map not found" not in automountmapdel.stderr'
```

Comment:
Please, deindent to follow coding style that is used for these playbooks.

Reply:
Done.